### PR TITLE
Update 09-debugging.md: exercise time

### PR DESCRIPTION
**Pull request by davidbenncsiro**
5 or 10 minutes to complete exercises seems reasonable for this episode. The first exercise could take awhile so perhaps 10 minutes is closer to the mark.

### CODE REVIEW

File: _episodes/09-debugging.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ we want to localize the failure to the smallest possible region of code:
     then concentrate on either the first or second half,
     and so on.
 
-2.  N things can interact in N! different ways,
+2.  N things can interact in N<sup>2</sup> different ways,
```
Comment (maxim-belkin) [inline _episodes/09-debugging.md]: Something went wrong... Could you please fix this PR... or submit a new one?

**Comment by davidbenncsiro** [comment]
Was it just that I changed the title? The changed version looks okay and the branch appears to have been merged. But let me know if I need to raise a new PR.

Apologies for the # of PRs in such a short time. This is I guess what happens when preparing in earnest for a teaching/demo session. :)

David
________________________________

File: _episodes/09-debugging.md (hunk 1 above)
Comment (davidbenncsiro) [reply _episodes/09-debugging.md]: Odd. Not sure what happened, so I'll submit a new PR as you suggest.

**Comment by davidbenncsiro** [comment]
Closing this one and opening a new one.

**Comment by davidbenncsiro** [comment]
Thanks for being so open to change.

**Pull request closed without merging**